hypershift-operator/controllers/machineconfigserver/machineconfigserver_controller.go

```diff
@@ -113,7 +113,11 @@ func (r *MachineConfigServerReconciler) Reconcile(ctx context.Context, req ctrl.
 			return ctrl.Result{}, err
 		}
 
-		switch mcs.Spec.IgnitionService.Type {
+		serviceType := mcs.Spec.IgnitionService.Type
+		if serviceType == "" {
+			serviceType = hyperv1.Route
```

Comment:
have we considered defaulting this on the API as in https://github.com/openshift/hypershift/blob/main/api/v1alpha1/nodepool_types.go#L50?

Reply:
That's definitely nice, will test it, thx!